src/meta_schedule/search_strategy/search_strategy.cc

```diff
@@ -21,6 +21,8 @@
 namespace tvm {
 namespace meta_schedule {
 
+TuningStatistics SearchStrategyNode::time_counter;
```

Comment:
Usually we don't use global variables, which could lead to various issues. Let's find some ways around :-)